scripts/source-maintenance.sh

```diff
@@ -26,6 +26,9 @@
 KeepGoing="no"
 # the actual name of the directive that enabled keep-going mode
 KeepGoingDirective=""
+#
+# the accepted astyle version. Can be overridden with --enable-astyle <version>
```

Comment:
Nit: Let's explain what "accepted" actually means in this context (and polish capitalization/wording).

```suggestion
# astyle version used for official sources; overwrite --with-astyle-version <version>
```

If other change requests are addressed, this change request may become obsolete or need adjustments (depending on how the other change requests are addressed).